topostats/grains.py

```diff
@@ -1019,10 +1019,10 @@ def get_individual_grain_crops(
     @staticmethod
     def vet_numbers_of_regions_single_grain(
         grain_mask_tensor: npt.NDArray,
-        class_region_number_thresholds: list[list[int, int, int]] | None,
+        class_region_number_thresholds: list[tuple[int, int, int]] | None,
```

Comment:
I often feel referring to the type in variable names is excessive and unnecessary, in `vet_class_sizes_single_grain` the option is simply `class_size_threholds` and I think we should be consistent throughout as its easier to understand code (at least I find it easier) when things that have the same meaning have the same name.

```suggestion
        class_region_thresholds: list[tuple[int, int, int]] | None,
```

Obviously use within the function needs updating but simple with the language server functionality to replace (although in my experience it rarely updates the docstring so that will need to be done manually).

Reply:
If I'm understanding you correctly, this variable name is intended to be the same style as `class_size_thresholds`, as in we are thresholding numbers of regions, hence `class_*region_number*_thresholds` but obviously this isn't very apparent.

I think `class_region_thresholds` would be too ambiguous, what are we thresholding about the class regions? Their size, height, number?

Any thoughts?

Reply:
Personally I find that rather abstract - how about `class_region_quantity_thresholds`?